ui/app/routes/vault/cluster/logout.js

```diff
@@ -21,7 +21,7 @@ export default Route.extend(ModelBoundaryRoute, {
     this.namespaceService.reset();
     this.console.set('isOpen', false);
     this.console.clearLog(true);
-    this.clearModelCache();
+    this.replaceWith('vault.cluster.auth', { queryParams: { redirect_to: '' } });
     this.flashMessages.clearMessages();
     this.permissions.reset();
     this.replaceWith('vault.cluster.auth');
```

Comment:
Not sure at all here, but noticed you are calling `replaceWith` twice now, wondering if this last one can be removed now you are calling it higher up? If so, should the new `queryParam`'ed one be moved to where this one is? (i.e. after the `clear`ing and `reset`ing)

Reply:
P.S. I suppose in other words ( 😆 ) , could you just add the `queryParam` to this last one instead? (obvs still removing the clearModelCache if that already happens when leaving the logout route)

Reply:
nice catch! the extra `replaceWith` snuck in during a merge conflict. i actually don't need the `queryParam` at all here, and including it broke a few tests. i removed the extra one and now they're back to passing. thank you!